### PR TITLE
figlet: add page

### PR DESCRIPTION
**Pull request by guojikai**

### CODE REVIEW

**Comment by guojikai** [comment]
@agnivade  I installed it via mac homebrew. I dont know whether it has a windows version or not but it has many versions covered almost every language. [https://github.com/search?p=2&q=figlet&ref=opensearch&type=Repositories](url)

**Comment by agnivade** [comment]
I see. Ok, then please remove the page under `linux/` and merge this page with that. And then we'll see how it turns out.

File: pages/common/figlet.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# figlet
+
+> Display large characters made up of ordinary screen characters.
```
Comment (agnivade) [inline pages/common/figlet.md]: I think the description of the previous page was pretty much to the point. IMO "ASCII banners" is a better and clearer description than saying "large characters".

 Lets keep the original description.

File: pages/common/figlet.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+
+- Use default settings:
```
Comment (agnivade) [inline pages/common/figlet.md]: We generally use descriptions which are action oriented. Like describing what the command does.

In this case, how about - `Generate an ascii banner for "Hello World"` ?

File: pages/common/figlet.md [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+
+`figlet {{Hello World}}`
+
+- To change the font, use the -f option:
+
+`figlet -f {{font}} {{Hello Wrld}}`
```
Comment (agnivade) [inline pages/common/figlet.md]: Wrld => typo

File: pages/common/figlet.md [hunk 4]
```diff
@@ -0,0 +1,15 @@
+# figlet
+
+> Display large characters made up of ordinary screen characters.
+
+- Use default settings:
+
+`figlet {{Hello World}}`
+
+- To change the font, use the -f option:
```
Comment (agnivade) [inline pages/common/figlet.md]: The usage of the `-f` option is pretty clear from the command itself. So no need to mention it again in the description.

How about - `Generate output by specifying a given font`

**Comment by waldyrious** [comment]
FYI, the removal of the "waiting" label was due to me making minor edits (typos, punctuation) to the comments above.

**Pull request closed without merging**